appveyor_before_build.ps1

```diff
@@ -13,15 +13,42 @@ function CopyCsharpPluginForPatching
     [System.IO.File]::Copy($csPluginCleanJar, $destinationFile)
 }
 
-if (!$env:APPVEYOR)
-{    
-    $LOCAL_DEBUG_RUN = 1
-}
+if (IsPRCABuild)
+{
+    echo "Running PR-CA build"
```

Comment:
The logic here was written by @dbolkensteyn in cmd files, so I simply transferred it to PowerShell. You would think it is trivial, but there are several differences between PowerShell and CMD that blew up on me ....